cygnus-ngsi/conf/agent_ngsi.conf.template

```diff
@@ -26,389 +26,437 @@
 # listening port the Flume source will use for receiving incoming notifications
-cygnusagent.sources.http-source.port = 5050
+cygnus-ngsi.sources.http-source.port = 5050
 # Flume handler that will parse the notifications, must not be changed
-cygnusagent.sources.http-source.handler = com.telefonica.iot.cygnus.handlers.NGSIRestHandler
+cygnus-ngsi.sources.http-source.handler = com.telefonica.iot.cygnus.handlers.NGSIRestHandler
 # URL target
-# cygnusagent.sources.http-source.handler.notification_target = /notify
-# Default service (service semantic depends on the persistence sink)
-# cygnusagent.sources.http-source.handler.default_service = default
-# Default service path (service path semantic depends on the persistence sink)
-# cygnusagent.sources.http-source.handler.default_service_path = /
-# Source interceptors, do not change
-cygnusagent.sources.http-source.interceptors = ts gi
+cygnus-ngsi.sources.http-source.handler.notification_target = /notify
+# default service (service semantic depends on the persistence sink)
+cygnus-ngsi.sources.http-source.handler.default_service = default
+# default service path (service path semantic depends on the persistence sink)
+cygnus-ngsi.sources.http-source.handler.default_service_path = /
+# source interceptors, do not change
+cygnus-ngsi.sources.http-source.interceptors = ts gi
 # TimestampInterceptor, do not change
-cygnusagent.sources.http-source.interceptors.ts.type = timestamp
-# GroupinInterceptor, do not change
-cygnusagent.sources.http-source.interceptors.gi.type = com.telefonica.iot.cygnus.interceptors.NGSIGroupingInterceptor$Builder
+cygnus-ngsi.sources.http-source.interceptors.ts.type = timestamp
+# GroupingInterceptor, do not change
+cygnus-ngsi.sources.http-source.interceptors.gi.type = com.telefonica.iot.cygnus.interceptors.NGSIGroupingInterceptor$Builder
 # Grouping rules for the GroupingInterceptor, put the right absolute path to the file if necessary
-# See the doc/design/interceptors document for more details
-cygnusagent.sources.http-source.interceptors.gi.grouping_rules_conf_file = /usr/cygnus/conf/grouping_rules.conf
+# see the doc/design/interceptors document for more details
+cygnus-ngsi.sources.http-source.interceptors.gi.grouping_rules_conf_file = /usr/cygnus/conf/grouping_rules.conf
 
 # ============================================
 # NGSIHDFSSink configuration
 # channel name from where to read notification events
-cygnusagent.sinks.hdfs-sink.channel = hdfs-channel
+cygnus-ngsi.sinks.hdfs-sink.channel = hdfs-channel
 # sink class, must not be changed
-cygnusagent.sinks.hdfs-sink.type = com.telefonica.iot.cygnus.sinks.NGSIHDFSSink
-# true if the grouping feature is enabled for this sink, false otherwise
-# cygnusagent.sinks.hdfs-sink.enable_grouping = false
+cygnus-ngsi.sinks.hdfs-sink.type = com.telefonica.iot.cygnus.sinks.NGSIHDFSSink
+# true or false, true applies the new encoding, false applies the old encoding.
+# cygnus-ngsi.sinks.hdfs-sink.enable_encoding = false
```

Comment:
It seems something stupid, but not in my head :D I would like commented parameters don't have a white space between `#` and the parameter. I.e.:

    #cygnus-ngsi.sinks.hdfs-sink.enable_encoding = false

Instead of:

    # cygnus-ngsi.sinks.hdfs-sink.enable_encoding = false

:)